assembl/tests/views_tests/test_graphql_idea_hashtags.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+from freezegun import freeze_time
+from graphql_relay import from_global_id
+
+from assembl.graphql.schema import Schema as schema
+from assembl.tests.fixtures.graphql import create_post_in_thread
+
+
+@freeze_time("2018-3-1")
+def test_idea_hashtags(graphql_request, request, test_session,
```

Comment:
You should also add a test for the edge case of only having a `#` with nothing else passed in. Eg.
```
post.body.entries[0].value = "The quick brown #Fox jumps over the lazy #".
```